### ISSUE TITLE
FXAA using pixel coverage for edge detection?

### ISSUE THREAD

**Issue by ToruTheRedFox** [gonetz/GLideN64#2606]
Since edge detection is now emulated, it can be used to provide edge data to the FXAA shader, which, with a few tweaks can provide a decent emulation of the N64's VI filter.

**Comment by Xii-Nyth** [comment]
Yes that would be great, only flaw is it cant apply aa withing a texture and only at edges, which certain methods which use depth buffer information can do already, but many games don't have full depth buffer information.

Perhaps I can use a custom model to create an edges where I have lines which are being aliased though 🤔 would take a lot of effort however.
#https://github.com/gonetz/GLideN64/issues/2673

**Comment by ToruTheRedFox** [comment]
AA within a texture is generally undesirable (and is why MLAA has dedicated passes and parameters to avoid it)
